### ISSUE TITLE
What are seqmap_file and seq_info

### ISSUE THREAD

**Issue by Spritea** [JonathonLuiten/HOTA-metrics#4]
Hi, this is a great job!

However, I found some parameters not familiar for me, could you please explain what files they correspond to, like in the MOT dataset, thanks!

They are seqmap_file and seq_info in the code [here](https://github.com/JonathonLuiten/HOTA-metrics/blob/d7210b0e89aec98813c34a9408451d54233353b3/hota_metrics/datasets/mot_challenge_2d_box.py#L33-L35).

**Comment by JonathonLuiten** [comment]
Happy to help! Thanks for asking!

Improving the documentation to make things like this clearer is something we're currently working on, but until then here's the overview.

Both the seqmap and seqinfo are MOTChallenge dataset specific files. The first is simply a file listing all of the sequences to evaluate. It can easily be written by hand, but I think currently it can only be found by downloading the official MOTChallenge toolkit (something to improve, thanks for pointing it out).

The seqinfo is actually a part of the MOTChallenge ground truth files. It is downloaded when downloading the GT files.

I provide a minimum example of data folder as I have it set up for MOT17 including all the gt data, seqmaps,  and an example tracker. This can just be 'unzipped' into the project repo and then should work. (e.g. after unzip paths should look like: HOTA-metrics/data/gt/mot_challenge/...)

[data.zip](https://github.com/JonathonLuiten/HOTA-metrics/files/5938624/data.zip)

Now your question was actually about the parameters in the code, which refer to where to find the information from seqmap and seqinfo.

Basically, SEQMAP_FILE allows you to directly specify a path to a seqmap file, instead of using the default path. If you're running standard MOT eval using my recommended file structure you can just leave as None.

SEQ_INFO allows one further step of customization, and not require one to read in any seqmap or seqinfo files, and just pass the required info directly into the function as a dict. This dict has keys as sequences names with values as sequence lengths. Again, this is optional, if set to None, it uses the default by reading in the files as described above.

This should make everything clear.

Regards,
Jonathon
